src/openms/include/OpenMS/ANALYSIS/ID/IDScoreSwitcherAlgorithm.h

```diff
@@ -124,8 +124,17 @@ namespace OpenMS
         // also already have the correct score set
         return;
       }
+
+      if (t.hasSuffix("_score"))
```

Comment:
this is kind of ugly in OpenMS. Would probably be better to deal with that on the parse / writer / data structure side. we just need to make sure to get rid of that stuff at some point in the future

Reply:
Yes definitely. It also makes it super hard to recover old scores since they are not registered at the run level anymore. But this would be solved in the new ID structure.